lua/wikis/commons/Infobox/Event.lua

```diff
@@ -0,0 +1,359 @@
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Array = Lua.import('Module:Array')
+local Class = Lua.import('Module:Class')
+local DateExt = Lua.import('Module:Date/Ext')
+local Game = Lua.import('Module:Game')
+local Json = Lua.import('Module:Json')
+local Logic = Lua.import('Module:Logic')
+local Namespace = Lua.import('Module:Namespace')
+local Page = Lua.import('Module:Page')
+local String = Lua.import('Module:StringUtils')
+local Table = Lua.import('Module:Table')
+local Variables = Lua.import('Module:Variables')
+
+local BasicInfobox = Lua.import('Module:Infobox/Basic')
+local LeagueIcon = Lua.import('Module:LeagueIcon')
+local Links = Lua.import('Module:Links')
+local Locale = Lua.import('Module:Locale')
+local ReferenceCleaner = Lua.import('Module:ReferenceCleaner')
+local TextSanitizer = Lua.import('Module:TextSanitizer')
+
+local Widgets = Lua.import('Module:Widget/All')
+local Accommodation = Widgets.Accommodation
+local Builder = Widgets.Builder
+local Cell = Widgets.Cell
+local Center = Widgets.Center
+local Chronology = Widgets.Chronology
+local Customizable = Widgets.Customizable
+local Header = Widgets.Header
+local Location = Widgets.Location
+local Organizers = Widgets.Organizers
+local SeriesDisplay = Widgets.SeriesDisplay
+local SeriesIcon = Widgets.SeriesIcon
+local Title = Widgets.Title
+local Venue = Widgets.Venue
+local Link = Lua.import('Module:Widget/Basic/Link')
+
+---@class InfoboxEvent: BasicInfobox
+local Event = Class.new(BasicInfobox)
+
+---@return Html
+function Event:createInfobox()
+	local args = self.args
+	self:_parseArgs()
+
+	local widgets = {
+		Header{
+			name = args.name,
+			image = args.image,
+			imageDark = args.imagedark or args.imagedarkmode,
+			size = args.imagesize,
+		},
+		Center{children = {args.caption}},
+		Title{children = 'Event Information'},
+		Cell{
+			name = 'Series',
+			content = {
+				SeriesDisplay{
+					displayManualIcons = Logic.readBool(args.display_series_icon_from_manual_input),
+					series = args.series,
+					abbreviation = args.abbreviation,
+					icon = args.icon,
+					iconDark = args.icondark or args.icondarkmode,
+					iconDisplay = self.iconDisplay
+				},
+				SeriesDisplay{
+					series = args.series2,
+					abbreviation = args.abbreviation2,
+				},
+			}
+		},
+		Customizable{
+			id = 'organizers',
+			children = {Organizers{args = args}},
+		},
+		Customizable{
+			id = 'sponsors',
+			children = {
+				Cell{name = 'Sponsor(s)', content = self:getAllArgsForBase(args, 'sponsor')},
+			}
+		},
+		Customizable{
+			id = 'gamesettings',
+			children = {
+				Builder{
+					builder = function()
+						local gameLink = Game.link{game = self.data.game, useDefault = false}
+						if not gameLink then return end
+						return Cell{name = 'Game', content = {Link{
+							link = gameLink,
+							children = {Game.name{game = self.data.game, useDefault = false}}
+						}}}
+					end
```

Comment:
IMO this should support multiple games, as it would be used between Series and League, right?
One event consists of multiple leagues, each league has one game, but the event has thus multiple games?

Reply:
fwiw the fighters template only supports one game as input
but sure we can add support for several games